src/sql-parser/src/parser.rs

```diff
@@ -392,6 +392,23 @@ impl Parser {
                     }
                 };
                 self.expect_token(&Token::RParen)?;
+                while self.consume_token(&Token::Period) {
+                    let field = match self.next_token() {
+                        Some(Token::Word(w)) => Some(w.to_ident()),
+                        Some(Token::Mult) => None,
```

Comment:
Ah, no, it's just a syntax error. (It's not _quite_ a syntax error in PostgreSQL because Yacc can't handle that, but _we_ can, and so I've made it such.) Thanks for catching!